### DISCUSSION TITLE
Couldn't find tauri 1.6.0 on npmjs.com

### DISCUSSION THREAD

**Discussion by amio** [tauri-apps/tauri#9137] [General]
I see tauri 1.6.0 has released https://beta.tauri.app/blog/tauri-1-6/
but the npmjs.com still shows 1.5.3 as latest version https://www.npmjs.com/package/@tauri-apps/api

**Comment by FabianLars** [comment]
I don't know why but we stopped caring about syncing the minor release for the last 1.x release. Technically there's no need for a 1.6 js package since nothing changed that would make it required, 1.5.3 still works for the rust 1.6.0 package.
